platform-core-module/src/main/java/com/softicar/platform/core/module/CoreI18n.java

```diff
@@ -767,6 +767,10 @@
 		.de("Reply-To");
 	I18n0 RESET_PASSWORD = new I18n0("Reset Password")//
 		.de("Passwort zurücksetzen");
+	I18n0 RETENTION_DAYS_OF_EXECUTIONS = new I18n0("Retention Days of Executions")//
```

Comment:
We should probably simplify the English label (and column name): `Execution Retention Days`... is easier to grasp. German translation could stay as-is.

Reply:
done